Makefile

```diff
@@ -61,7 +61,7 @@ mock-generate:
 
 	# Generate mocks for imported protobuf clients too
 	mockery --config=mocks/.mockery.yaml --name=IPsecClient --srcpkg=github.com/opiproject/opi-api/security/v1/gen/go
-	mockery --config=mocks/.mockery.yaml --name=InventorySvcClient --srcpkg=github.com/opiproject/opi-api/common/v1/gen/go
```

Comment:
are we also missing new line here for `mocks/InvClient.go` ? wdyt ?

Reply:
Its higher up in the Makefile
mock-generate:
	@echo "  >  Starting mock code generation..."
	# Generate mocks for exported interfaces
	mockery --config=mocks/.mockery.yaml --name=Connector --dir=grpc
	mockery --config=mocks/.mockery.yaml --name=InvClient --dir=inventory